src/plugins/intel_cpu/src/emitters/tpp/common/utils.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+
+#include "emitters/utils.hpp"
+#include "libxsmm.h"
+
+namespace ov::intel_cpu::tpp {
```

Comment:
Should we use `utils` namespace for these helpers (just to be aligned with other utils* files)?

Reply:
utils is added to namespace, thanks Vladislav!